tests/test_ops.py

```diff
@@ -85,6 +85,9 @@ def test_ops_layout_transparent():
         'relu.float32',
         'prelu.float32',
         'sigmoid.float32',
+        'sub.float32',
+        'abs-sqrt.float32',
```

Comment:
I am curious about if it's possible to have `sqrt` and `power` models by themselves. Dedicated models containing only the specific operator could be helpful when we debugging.

Reply:
`m.genInput()` in `end2end_test` generates random numbers. However, square root from negative real numbers does not produce real number (results in NaN instead), thus I had to either use complex data type or get absolute values before applying square root.

Same with the power. If some negative power is applied to zero, that has the same effect as division by zero. I have also had some troubles with large exponentiation, like Pow(100, 1000), so I decided to clamp input values with ReLU6.

If you happen to know any other option to limit input values, please let me know.

Reply:
Oh, we need to add options to [`shrub`](https://github.com/jackwish/shrub/blob/master/shrub/network.py#L99). But that will be a TODO. Thanks for the explanation.